internal/uvm/create_lcow.go

```diff
@@ -102,6 +119,14 @@ func CreateLCOW(opts *OptionsLCOW) (_ *UtilityVM, err error) {
 			opts.RootFSFile = "rootfs.vhd"
 		}
 	}
+	if opts.ForwardStdout == nil {
+		val := false
```

Comment:
Why false by default?

Reply:
Oh interesting I didn't realize we didn't forward it today.

Reply:
This preserves existing behavior of forwarding stderr from GCS (because it has logrus messages), but not stdout. If GCS doesn't output anything to stdout, changing this wouldn't break anything, but if it does, we'd try to parse it as a logrus message.